app/pages/solutions-list/filterType/covid19/data/covid19-solutions.json

```diff
@@ -10,7 +10,11 @@
       },
       "covid19": {
         "title": "How this Catalogue Solution can help with coronavirus",
```

Comment:
since the title is the same for every solution can we move it to the manifest?

Reply:
The idea is that they could be changed individually if needed